modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java

```diff
@@ -441,13 +444,18 @@ private void _setUpJSONFactoryUtil() {
 	}
 
 	private void _setUpLanguageUtil() {
-		mockStatic(LanguageUtil.class);
+		LanguageUtil languageUtil = new LanguageUtil();
+
+		Language language = Mockito.mock(Language.class);
 
-		when(
-			LanguageUtil.isAvailableLocale(Matchers.any(Locale.class))
-		).thenReturn(
+		Mockito.doReturn(
 			Boolean.TRUE
+		).when(
+			language
+		).isAvailableLocale(
+			Matchers.any(Locale.class)
 		);
+		languageUtil.setLanguage(language);
```

Comment:
@JanisZhang For format, usually we keep an empty line here (before line 458).

Reply:
@dantewang Got it. I will force push, thanks!